frontend/packages/ceph-storage-plugin/integration-tests/tests/4-pools/multiple-pool.scenario.ts

```diff
@@ -0,0 +1,111 @@
+import { browser } from 'protractor';
+import { execSync } from 'child_process';
+import { click } from '@console/shared/src/test-utils/utils';
+import { NS, CLUSTER_STATUS, SECOND } from '../../utils/consts';
+import {
+  prepareStorageClassForm,
+  allowExpand,
+  poolDropdownButton,
+  createPoolDropdown,
+  createPool,
+  poolForm,
+  poolStatusCheck,
+  poolMessage,
+  POOL_STATUS,
+  finishButton,
+  dropdownPoolName,
+  poolDescription,
+  modalPresence,
+} from '../../views/multiple-pool.view';
+
+const cephValue = JSON.parse(execSync(`kubectl get cephCluster -n ${NS} -o json`).toString());
+const cephStatus = cephValue.items[0];
+
+describe('Check availability of Ceph cluster', () => {
+  if (!cephStatus) {
+    it('Should state that Ceph cluster doesnt exist', () => {
+      expect(cephStatus).toBeUndefined();
+    });
+  } else {
+    it('Should state that Ceph cluster exist', () => {
+      expect(cephStatus).toBeDefined();
+    });
+  }
+});
+
+if (cephStatus && cephStatus.status.phase && cephStatus.status.phase !== CLUSTER_STATUS.READY) {
+  describe('Check for pool creation if ceph cluster is in progress', () => {
+    beforeAll(async () => {
+      prepareStorageClassForm('openshift-storage.rbd.csi.ceph.com');
```

Comment:
Don't we need to test `fs` as well?

Reply:
Since the pool creation is for `rbd` provisioner, I wrote test cases for that, we can add test cases for `fs` too, but those are not that imp.